src/utilities/array.ts

```diff
@@ -38,8 +38,15 @@ export const sortArray = (inputArray: Array<any>, sortOptions: Array<SortProps>)
                 valueA = sortOption.sortFunction(a);
                 valueB = sortOption.sortFunction(b);
             } else {
-                valueA = a[sortOption.column];
-                valueB = b[sortOption.column];
+                let splits = sortOption.column.split('.');
+                let resultA = a;
+                let resultB = b;
+                for (let i = 0; i < splits.length; i++) {
```

Comment:
same as for the groupinng operation, won't this just set valueA and ValueB to the deepest value?

Reply:
Deepest FOUND.